hack/local-development/local-garden/run-kube-apiserver

```diff
@@ -13,7 +13,7 @@ LABEL=${1:-local-garden}
 
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 
-IMAGE=k8s.gcr.io/kube-apiserver:v1.17.9
+IMAGE=k8s.gcr.io/kube-apiserver:v1.19.7
```

Comment:
Can we go directly to 1.20.2?

Reply:
Thanks for the suggestion @rfranzke. I tested it successfully and updated the PR.